prometheus_juju_backup_all_exporter/__main__.py

```diff
@@ -42,41 +35,8 @@ def main() -> None:
     root_logger.setLevel(logging.getLevelName(config.level))
 
     exporter = Exporter(config.port)
-    exporter.register(
-        BackupStatsCollector(
-            config,
-            BackupDurationMetric(
-                "juju_backup_all_command_duration_seconds",
-                "Length of time the charm-juju-backup-all backup command took.",
-                labels=["status_ok", "result_code"],
-            ),
-            BackupStatusOKMetric(
-                "juju_backup_all_command_ok_info",
-                "Indicates whether or not the charm-juju-backup-all backup command was a success.",
-                labels=["result_code"],
-            ),
-        )
-    )
-    exporter.register(
-        BackupStateCollector(
-            config,
-            BackupFailedMetric(
-                "juju_backup_all_backup_failed_total",
-                "The number of failed backups.",
-                labels=[],
-            ),
-            BackupPurgedMetric(
-                "juju_backup_all_backup_purged_total",
-                "The number of purged backups.",
-                labels=[],
-            ),
-            BackupCompletedMetric(
-                "juju_backup_all_backup_completed_total",
-                "The number of completed backups.",
-                labels=[],
-            ),
-        )
-    )
+    exporter.register(BackupStatsCollector(config))
+    exporter.register(BackupStateCollector(config))
```

Comment:
non-blocker: It'd help with readability if these two classes had more distinct names (`BackupStatsCollector`,`BackupStateCollector`). I can't think of anything right now but if you can come up with any sensible alternative name that'd be great. If not, disregard.

Reply:
Updated